addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/JsFunctionScanRule.java

```diff
@@ -176,7 +176,7 @@ private void loadPayload() {
     private static void addPattern(String line, List<Pattern> list) {
         // Strip leading $, it's optionally included in the assembled patterns
         line = line.replace("$", "");
-        list.add(Pattern.compile("\\b\\$?" + line + "\\b", Pattern.CASE_INSENSITIVE));
+        list.add(Pattern.compile("\\b\\$?" + line + "\\(", Pattern.CASE_INSENSITIVE));
```

Comment:
It will now miss, e.g. `eval (`.

Reply:
I guess I could add optional white space to the regex. 5 char seem like enough? JS doesn't care about white space or wrapping right?